locales/sw/auth.php

```diff
@@ -14,5 +14,5 @@
 return [
     'failed'   => 'Hii hati tambulishi hailingani na rekodi zetu.',
     'password' => 'Nenosiri lililotolewa si sahihi.',
-    'throttle' => 'Majaribio mengi sana ya kuingia. Tafadhali jaribu tena katika :sekunde sekunde.',
+    'throttle' => 'Majaribio mengi sana ya kuingia. Tafadhali jaribu tena katika sekunde :sekunde.',
```

Comment:
*:placeholders* must not be translated!
*:seconds* is a placeholder and it will be replaced by a number.